components/motor/gpio/mathutils.go

```diff
@@ -48,17 +41,10 @@ func goForMath(maxRPM, rpm, revolutions float64) (float64, time.Duration) {
 // goForMath calculates goalPos, goalRPM, and direction based on the given GoFor rpm and revolutions, and the current position.
 func encodedGoForMath(rpm, revolutions, currentPos, ticksPerRotation float64) (float64, float64, float64) {
 	direction := sign(rpm * revolutions)
```

Comment:
we could return early when direction == 0 here as well, but the logic below already handles the `rpm == 0` and `revolutions == 0` case, so maybe fine as is